src/server.js

```diff
@@ -69,23 +70,33 @@ const initServer = async function (db, config) {
         }, 'stdout']
       }
     }
-  }])
+  }, {
 
-  // add a health endpoint on /
-  server.route({
-    method: 'GET',
-    path: '/',
-    handler: async (request, h) => {
-      if (!(await db.isConnected())) {
-        return h.response({
-          statusCode: 500,
-          error: 'Internal Server Error',
-          message: 'Database not connected' }).code(500)
+  // add a health endpoint on Database being available/
+    plugin: Health,
+    options: {
+      path: '/health', //Health route path
```

Comment:
Please look at other APIs that import routes.js files and align the API accordingly